source/administration/changelog.md

```diff
@@ -223,6 +223,10 @@ Multiple setting options were added to `config.json`. Below is a list of the add
  - Added ``KVCompareAndSet(key string, old []byte, new []byte)`` to Plugin API to add support for transactional semantics with KV Store in plugin framework.
 
 ### Known Issues
+  - Channels with large message history may have performance issues. This is scheduled to be resolved in v5.13.1.
```

Comment:
Nevermind, these were added to 5.12.5 so maybe it's not a big deal to add as known issues. Will let you decide. For some reason I thought they were in 5.13 only